pyconfig.h.in

```diff
@@ -569,7 +578,7 @@
 #undef HAVE_LANGINFO_H
 
 /* Defined to enable large file support when an off_t is bigger than a long
-   and long long is at least as big as an off_t. You may need to
+   and long long is available and at least as big as an off_t. You may need to
```

Comment:
This line should be reverted.

Reply:
Hummm, this is the result of running `autoreconf`. Maybe I have an outdated version of the configure sources....

Reply:
@sir-sigurd Yeah, just confirmed that running `autoreconf` on maste produces this diff:

```
diff --git a/pyconfig.h.in b/pyconfig.h.in
index ffdc49514e..20cc901e47 100644
--- a/pyconfig.h.in
+++ b/pyconfig.h.in
@@ -569,7 +569,7 @@
 #undef HAVE_LANGINFO_H
 
 /* Defined to enable large file support when an off_t is bigger than a long
-   and long long is at least as big as an off_t. You may need to
+   and long long is available and at least as big as an off_t. You may need to
    add some flags for configuration and compilation to enable this mode. (For
    Solaris and Linux, the necessary defines are already defined.) */
 #undef HAVE_LARGEFILE_SUPPORT
```

this should be fixed in a different PR.

Reply:
I'll create PR to fix it.